tests/test_edgeql_insert.py

```diff
@@ -1024,6 +1024,76 @@ class TestInsert(tb.QueryTestCase):
             ]
         )
 
+    async def test_edgeql_insert_default_05(self):
+        # Issue #730
+        await self.con.execute(r'''
+            # The 'number' property is supposed to be
+            # self-incrementing and read-only.
+            INSERT test::DefaultTest8;
+            INSERT test::DefaultTest8;
+            INSERT test::DefaultTest8;
+        ''')
+
+        await self.assert_query_result(
+            r'''
+                WITH MODULE test
+                SELECT DefaultTest8.number;
+            ''',
+            {1, 2, 3}
+        )
+
+    @test.xfail('''
+        Presumably, forcing a specific sequence value in an INSERT
```

Comment:
This is a wrong assumption to make.  Sequences are a standalone resource in the database and having a serial column basically means you have the implicit default of `nextval('my_sequence')`.  Inserting explicit values does not affect the underlying sequence at all.

Reply:
OK, I've removed the 2 tests that `INSERT` a specified value instead of relying on the automatic sequence as they don't really test anything in particular.